### PR TITLE
9.0 assets fix

### PR DESCRIPTION
**Pull request by KangOl**

### CODE REVIEW

File: openerp/addons/base/ir/ir_qweb.py [hunk 1]
```diff
@@ -1326,7 +1327,23 @@ def css_message(self, message):
...
+                ira_ids = self.registry['ir.attachment'].search(self.cr, self.uid, assets_domain)
+                if ira_ids and len(ira_ids) == len(assets):
+                    ira_records = self.registry['ir.attachment'].browse(self.cr, self.uid, ira_ids)
+                    for ira_record in ira_records:
+                        stylesheet = next(stylesheet for stylesheet in self.stylesheets if stylesheet.html_url % stylesheet.url == ira_record.url)
+                        if stylesheet.last_modified > Datetime.from_string(ira_record['__last_update']):
+                            ira_records.unlink()
+                            return False
+                        stylesheet._content = ira_record.datas.decode('base64').decode('utf8'),
+                    return True
```
Comment (KangOl) [inline openerp/addons/base/ir/ir_qweb.py]: dedent.

**Comment by dmo-odoo** [comment]
@sle-odoo @KangOl @odony al wants to merge this. It's ok for me, do you have an objection ? Without any answer, I'll merge it tomorrow.

**Comment by KangOl** [comment]
You need to squash last 2 commits.

**Comment by sle-odoo** [comment]
last time i checked the patch was working but i did not retry it
also, i'm not sure @odony will like the test case

**Comment by dmo-odoo** [comment]
I didn't have time to merge it last week. @odony I will merge it tomorrow or, at the latest, next monday.
If you have some comments to make regarding the test case like @sle-odoo is suggesting, now would be a good time to do so... ;-)

**Comment by dmo-odoo** [comment]
Merged at a62597e1fa0455f4757449b6da58f80ec047b4af.

**Pull request closed without merging**